### ISSUE TITLE
Host latest release and documentation links on github pages

### ISSUE THREAD

**Issue by morganwillcock** [adventuregamestudio/ags#616]
With a logo and some CSS we could host the latest release and documentation links directly on github, to ensure that:

- download links automatically stay in sync with actual releases
- links to documentation are current
- linked documentation is editable by the people doing the releasing (no old manuals and incorrect wiki pages)

Linking the release seems pretty trivial using Jekyll, so it is really a question of styling for the page (probably just a logo will do, see #570) and whether this is deemed a positive step.

**Comment by ericoporto** [comment]
Github has an inbuilt Jekyll CI, it's not latest version and doesn't have everything, and the logs are slightly mysterious (the repository owner receives an e-mail when failing). Using a CI can enable using a different Jekyll version, but if one does use a CI, the rendering doesn't HAVE to be through Jekyll. Github has a lot of free apps on marketplace.

Since we have the manual pages already converted to Markdown, switching to other things that render markdown are theoretically easy - see all the sphinx journey, meaning some problems may arrive. Note the manual release can be tagged as also the ags-manual wiki, though the wiki requires being manipulated through an external git tool manipulated or transferred to a repository of it's own, also support tagging.

For now I understand you are just suggesting have a webpage that mimics the content of Github Release/AGS Forum releases that mimics a project webpage and includes links to documentation?

**Comment by morganwillcock** [comment]
> For now I understand you are just suggesting have a webpage that mimics the content of Github Release/AGS Forum releases that mimics a project webpage and includes links to documentation?

Yes I mean to, for now, just use Github Pages directly. So just basic information and the download links. I've tested getting the release URLs and it was pretty straightforward:
https://help.github.com/articles/repository-metadata-on-github-pages/
...the only downside is that the error log for a build would only show in the repository settings.

Potentially we can share the CSS / logo with the Sphinx build.

**Comment by ericoporto** [comment]
Just a comment, to create a webpage under directly the organization name, create a repository in the organization called `adventuregamestudio.github.io` (yes, with dots in the name). Make sure to not include subfolders that conflict with any repository name under the organization. If you use github pages on that repository, you will be able to access it directly through https://adventuregamestudio.github.io

**Comment by morganwillcock** [comment]
I don't have the permissions to create the repository, but since website is still linking to the old version I'd like to give this a try.

@ivan-mogilko would this be okay with you?

**Comment by ivan-mogilko** [comment]
@morganwillcock is this covered now or something is left?

**Comment by morganwillcock** [comment]
This was to was to host a page at https://adventuregamestudio.github.io/ that functioned as a download portal (linked to release assets but presented in a user friendly way, probably just the cup logo and the links). So it is slightly controversial, but as far as I know no-one here has access to update a release at adventuregamestudio.co.uk.

**Comment by ivan-mogilko** [comment]
I have an access to upload new releases on adventuregamestudio.co.uk, admin gave me permissions onto their fileserver.

**Comment by morganwillcock** [comment]
Yes, I think so.

**Comment by ivan-mogilko** [comment]
@morganwillcock what is the plan on this? Above you mentioned that you don't have permissions to create a new repository (in this project), is this what was restricting the progress? I might have missed this part.

**Comment by morganwillcock** [comment]
I'm pretty sure that I have the permissions required to create the repository. What I described could just be a static web page, something like this:

\<LOGO> Adventure Game Studio

- [Latest AGS release](https://github.com/adventuregamestudio/ags/releases/latest)
- [Latest AGS manual release](https://github.com/adventuregamestudio/ags-manual/releases/latest)
- [Online manual](https://adventuregamestudio.github.io/ags-manual/)
- [Development builds](https://cirrus-ci.com/github/adventuregamestudio/ags)

It could also be fancier and list the versions, including querying for pre-release status. This would likely require the setup of triggers to rebuild the web page when releases are created.
